scripts/create_polygons.py

```diff
@@ -41,14 +42,17 @@ def get_pixel_count(file_path: str) -> int:
 
 
 def main() -> None:  # pylint: disable=too-many-locals
-    logger = get_log()
+    start_time = time_in_ms()
+    node_id = os.getenv("ARGO_NODE_ID")
 
     parser = argparse.ArgumentParser()
     parser.add_argument("--source", dest="source", nargs="+", required=True)
     arguments = parser.parse_args()
     source = arguments.source
 
     source = format_source(source)
+    get_log().info("create_polygons_start", nodeId=node_id, source=source)
```

Comment:
`nodeId` needs to be added to *every* log message not just the first one. I think there is something like `get_log().add_keys(?)`

I wonder if it is also just the `hostname` key that is common to most of our other logs?

Reply:
Yes my bad. `get_log().add_keys()` allows you to permanently add a key to your logger instance?